src/main/java/com/gradle/CustomBuildScanEnhancements.java

```diff
@@ -306,6 +306,10 @@ public void execute(BuildResultAdapter buildResult) {
                     addCustomValueAndSearchLink(develocity, "CI run", value));
                 envVariable("GITHUB_HEAD_REF", providers).filter(value -> !value.isEmpty()).ifPresent(value ->
                     buildScan.value("PR branch", value));
+                envVariable("GITHUB_ACTION", providers).ifPresent(value ->
+                        addCustomValueAndSearchLink(develocity, "CI step id", value));
+                envVariable("GITHUB_JOB", providers).ifPresent(value ->
+                        addCustomValueAndSearchLink(develocity, "CI job id", value));
```

Comment:
I would remove `id`.

```suggestion
                envVariable("GITHUB_ACTION", providers).ifPresent(value ->
                        addCustomValueAndSearchLink(develocity, "CI step", value));
                envVariable("GITHUB_JOB", providers).ifPresent(value ->
                        addCustomValueAndSearchLink(develocity, "CI job", value));
```

Reply:
I'm happy to do that, however the values are only getting us `id` values, not names. If there's no id, it will show the name of whatever is being executed, if it's an external action, it'll show the name of that, for example something like `_gradle_actions_setup-gradle`, or `_run` and if there's muliple steps of the same thing, it'll show `_run_4`. I'm happy to drop the `id`, but this makes it clearer.

Reply:
We don't use `id` in any of the other values. Even when we capture `GITHUB_RUN_ID`, the custom value and link are named `CI run`. To me, appending `id` adds no value here and would introduce an inconsistency with all other values. 

Also keep in mind that the link gets `build scans` appended to it, so how it is now we'd have `CI job id build scans`.

Reply:
Yes, but there it's evident it's going to be an `id`, here one could expect to get the `name` for example, which you can't.

Reply:
I don't necessarily disagree with you, but I still think remaining consistent and not using `id` should be the preferred.